### ISSUE TITLE
Open Source?

### ISSUE THREAD

**Issue by nerdkid93** [GlennRicaud/data-toolbox-app#27]
Hello,

I was pointed in your direction by the Enonic team for using your app in XP, but I just noticed that there's no code in this repo. It seems to be housed in another private repository, yet you have this published as `Apache-2.0`. Would you consider opening up the actual source code for this application?

Thanks

**Comment by GlennRicaud** [comment]
Hello,

Yes, it is true that some libraries of the application are not open source and some libraries are located in another account.
I am thinking about regrouping it all under this account and opening it up yes.

Would you be interested in contributing? Or is there something you need in the meantime (FYI: The documentation issue you mentioned has now been fixed).

**Comment by nerdkid93** [comment]
I had just noticed the documentation link issue and had planned on submitting a PR to fix the links (esp. since it's [Hacktoberfest](https://hacktoberfest.digitalocean.com/)). When I noticed that most of the code is actually in the base package, I searched through the GitHub issues to see if anyone had brought this up before. When I didn't find anything, I figured it wouldn't hurt to ask if this structure was intentional or not.

So no, I don't have anything I need. :smile: I just wanted to be a good OSS community member and contribute back.
